apis/coralogix/v1alpha1/webhook_suite_test.go

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
I believe you should add `./apis/` to the unit-test makefile target, I changed it from `./...` in one of my PRs, so it won't run the e2e tests.